src/common/sanity.ts

```diff
@@ -88,10 +88,11 @@ export const sanityLanguageId = (locale: string): string => {
 };
 
 export const project = "ajwvhvgo";
-export const dataset = flags.cmsPreview ? "apps-preview" : "apps";
+export const dataset = "apps-preview";
```

Comment:
I have fixed the dataset as “apps-preview” for now just for testing because “apps” is querying the prod version.